ODPC.Server/Features/Organisaties/AlleOrganisaties/OrganisatiesController.cs

```diff
@@ -1,16 +1,28 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using System.Text.Json.Nodes;
+using Microsoft.AspNetCore.Mvc;
 using ODPC.Apis.Odrc;
 
 namespace ODPC.Features.Organisaties.AlleOrganisaties
 {
     [ApiController]
-    public class OrganisatiesController : ControllerBase
+    public class OrganisatiesController(IOdrcClientFactory clientFactory) : ControllerBase
     {
         [HttpGet("api/v1/organisaties")]
-        public IActionResult Get([FromQuery] string? page) => Ok(new PagedResponseModel<WaardelijstResponseModel>
+        public async Task<IActionResult> Get([FromQuery] string? page, CancellationToken token)
         {
-            Results = OrganisatiesMock.Organisaties.Values.ToList(),
-            Count = OrganisatiesMock.Organisaties.Count,
-        });
+            // organisaties ophalen uit het ODRC
+            using var client = clientFactory.Create("Organisaties ophalen");
+            var json = await client.GetFromJsonAsync<PagedResponseModel<JsonNode>>("/api/v1/organisaties?page=" + page, token);
+            if (json != null)
+            {
+                json.Previous = GetPathAndQuery(json.Previous);
+                json.Next = GetPathAndQuery(json.Next);
+            }
+            return Ok(json);
+        }
+
+        private static string? GetPathAndQuery(string? url) => Uri.TryCreate(url, UriKind.RelativeOrAbsolute, out var uri)
```

Comment:
nitpick: kunnen we dit extraheren naar een gedeelde functie?

Reply:
In overleg @felixcicatt afgesproken dat we zo laten tot thema's erbij komen en dan kijken naar gedeelte functies.